fe/fe-core/src/main/java/com/starrocks/connector/ConnectorPartitionTraits.java

```diff
@@ -543,57 +543,21 @@ PartitionKey createEmptyKey() {
-            Set<String> result = Sets.newHashSet();
-            for (BaseTableInfo baseTableInfo : baseTables) {
-                if (!baseTableInfo.getTableIdentifier().equalsIgnoreCase(baseTable.getTableIdentifier())) {
-                    continue;
-                }
-                Optional<ConnectorMetadata> connectorMetadata = GlobalStateMgr.getCurrentState().getMetadataMgr().
-                        getOptionalMetadata(baseTable.getCatalogName());
-                if (!connectorMetadata.isPresent()) {
-                    LOG.error("Get paimon connectorMetadata failed : {}", baseTable.getCatalogName());
-                    throw new RuntimeException("Get paimon connectorMetadata failed :" + baseTable.getCatalogName());
-                }
-                Map<String, MaterializedView.BasePartitionInfo> partitionVersionMap =
-                        context.getBaseTableRefreshInfo(baseTableInfo);
-                Set<String> partitions = Sets.newHashSet(getPartitionNames());
-                long mvLatestSnapShotID = Long.MIN_VALUE;
-                for (Map.Entry<String, MaterializedView.BasePartitionInfo> entry : partitionVersionMap.entrySet()) {
-                    if (entry.getValue() != null) {
-                        mvLatestSnapShotID = Math.max(mvLatestSnapShotID, entry.getValue().getVersion());
-                    }
-                    // If there are partitions deleted, return all latest partitions.
-                    if (!partitions.contains(entry.getKey())) {
-                        result.addAll(partitions);
-                        LOG.info("Get paimon updated partition names {}, partition {} has been deleted, return all.",
-                                baseTables, entry.getKey());
-                        return result;
-                    }
-                }
-                for (String part : partitions) {
-                    if (!partitionVersionMap.containsKey(part)) {
-                        result.add(part);
-                    }
-                }
-                ConnectorMetadata metadata = connectorMetadata.get();
-                List<PartitionInfo> changedPartitionInfo = metadata.getChangedPartitionInfo(baseTable,
-                        mvLatestSnapShotID);
-                for (PartitionInfo partitionInfo : changedPartitionInfo) {
-                    com.starrocks.connector.paimon.Partition info =
-                            (com.starrocks.connector.paimon.Partition) partitionInfo;
-                    // Change log record partition which has been deleted.
-                    if (!partitions.contains(info.getPartitionName())) {
-                        continue;
-                    }
-                    result.add(info.getPartitionName());
-                }
-            }
-            LOG.debug("Get updated partition name of paimon table result: {}", result);
-            return result;
+        public List<PartitionInfo> getPartitions(List<String> partitionNames) {
```

Comment:
The core reason is that some methods are not overwritten? please add some descriptions in the PR's comment.